### PR TITLE
Minor tweaks to insure compatibility with the tester script

### PR DESCRIPTION
**Pull request by AlexanderVi**
1.added format parameter(BAM/CRAM) 
2.Removed check for start/end read positions

### CODE REVIEW

File: src/main/java/uk/ac/ebi/ena/sra/client/TicketClient.java [hunk 1]
```diff
@@ -170,6 +180,9 @@ private static Report testTicketForQuery(TicketResponse r, Query query) throws I
     }
 
     private static TicketResponse getTicket(URL url, boolean printTicket) throws IOException, EndpointException {
+
+	    //System.out.println(url);
```
Comment (vadimzalunin) [inline src/main/java/uk/ac/ebi/ena/sra/client/TicketClient.java]: please remove

File: src/main/java/uk/ac/ebi/ena/sra/client/TicketClient.java [hunk 2]
```diff
@@ -117,8 +116,19 @@ private static void diagnostics(String id, Query query, String name, Provider pr
...
+    {        
+        String url = String.format("%s%s?format=%s&referenceName=%s", base, accession, format==Format.BAM?"BAM":"CRAM", query.sequence);
+		if( query.start > 0  )
+		{
+	        url = String.format( url+"&start=%d", query.start );
+	        if( query.end > 0 )
+	       		url = String.format( url+"&end=%d", query.end );
+	    }
+
+	    //System.out.println("URL:"+url);
```
Comment (vadimzalunin) [inline src/main/java/uk/ac/ebi/ena/sra/client/TicketClient.java]: please remove

File: src/main/java/uk/ac/ebi/ena/sra/client/TicketClient.java [hunk 3]
```diff
@@ -103,7 +102,7 @@ private static void diagnostics(String id, Query query, String name, Provider pr
         String sURL = null;
         try {
             System.out.printf("\t\t%-10s", name);
-            sURL = formatURL(provider.base, accession, query);
+            sURL = formatURL(provider.base, accession, query, id.contains("BAM")?Format.BAM:Format.CRAM);
```
Comment (vadimzalunin) [inline src/main/java/uk/ac/ebi/ena/sra/client/TicketClient.java]: case sensitive check, ideally should check if name ends with '.bam' or '.cram', perhaps String toLowerCase+endsWith or compareIgnoreCase or just regexp

**Comment by AlexanderVi** [comment]
will submit another PR with cumulative changes

**Pull request closed without merging**